src/main/resources/tools.json

```diff
@@ -4,7 +4,7 @@
       "version": "2.0.7",
       "versionCmd": "version --client",
       "versionExtractRegExp": "odo v(\\d+[\\.\\d+]*(-.*)?)\\s.*",
-      "versionMatchRegExpr": "0\\..*",
+      "versionMatchRegExpr": "2.0.7",
```

Comment:
not sure about this. are we need to change that on every new version of do ? shouldn't be something more like "2\\..*" for all 2.X versions

Reply:
No because odo does not offer backward and forward compatibility so better stick with the release the plugin was built against